crates/astria-bridge-contracts/src/snapshots/astria_bridge_contracts__tests__memo_snapshot.snap

```diff
@@ -0,0 +1,5 @@
+---
+source: crates/astria-bridge-contracts/src/lib.rs
+expression: "memo_to_json(&event, rollup_block_number, rollup_withdrawal_event_id).unwrap()"
+---
+{"rollupBlockNumber":"999","rollupWithdrawalEventId":"event ID","rollupReturnAddress":"0x00000000000000000000000000000000000000000000000000000000000003e7","memo":"a memo"}
```

Comment:
Should we make this look more like a normal evm address?

Reply:
Done in [b1d2e0a](https://github.com/astriaorg/astria/pull/1753/commits/b1d2e0a3949ce607ef7ca7097f79f412d7843436).